singularity/covid19_base.singularity

```diff
@@ -79,7 +79,7 @@ From: nvidia/cuda:10.2-base-ubuntu18.04
   #/opt/miniconda/bin/conda install -c bioconda minimap2=2.17 -y
   
   #install cutadapt
-  conda install -c bioconda cutadapt=1.18 -y
+  conda install -c bioconda cutadapt=1.18 pandoc -y
```

Comment:
> Hi Vang (@biocyberman). Can you set the specific version of pandoc used too? I have tried to be explicit with versions throughout the definition file. Why install with the same command as cutadapt? Not that it matters much though

Now I see it :) Just didn't want to start a newline. Pandoc via conda is likely to have newer version than Ubuntu stock version.

Reply:
Okay. I will figure out the current version and explicitly install that and commit again. Also with new guppy version. Not going to uptick version though, to avoid breaking anything now